### PR TITLE
Add index for table ReleaseHistory

### LINKED ISSUE #4544: ReleaseHistory表发现SQL没走索引，导致全表查询
ReleaseHistoryRepository类中的方法
Page<ReleaseHistory> findByPreviousReleaseIdAndOperationOrderByIdDesc(long previousReleaseId, int operation, Pageable pageable);
没有走索引，导致全表查询，产生慢查询

### PR DESCRIPTION
**Pull request by mghio**
## What's the purpose of this PR

Add index for table ReleaseHistory.

## Which issue(s) this PR fixes:
Fixes #4544

Follow this checklist to help us incorporate your contribution quickly and easily:

- [x] Read the [Contributing Guide](https://github.com/ctripcorp/apollo/blob/master/CONTRIBUTING.md) before making this pull request.
- [x] Write a pull request description that is detailed enough to understand what the pull request does, how, and why.
- [ ] Write necessary unit tests to verify the code.
- [x] Run `mvn clean test` to make sure this pull request doesn't break anything.
- [x] Update the [`CHANGES` log](https://github.com/ctripcorp/apollo/blob/master/CHANGES.md).

### CODE REVIEW

File: scripts/sql/apolloconfigdb.sql [hunk 1]
```diff
@@ -345,7 +345,8 @@ CREATE TABLE `ReleaseHistory` (
   PRIMARY KEY (`Id`),
   KEY `IX_Namespace` (`AppId`,`ClusterName`,`NamespaceName`,`BranchName`),
   KEY `IX_ReleaseId` (`ReleaseId`),
-  KEY `IX_DataChange_LastTime` (`DataChange_LastTime`)
+  KEY `IX_DataChange_LastTime` (`DataChange_LastTime`),
```
Comment (nobodyiam) [inline scripts/sql/apolloconfigdb.sql:348]: Please add a folder `v200-v210` in [delta](https://github.com/apolloconfig/apollo/tree/master/scripts/sql/delta) and add a upgrade script there.

Comment (mghio) [reply scripts/sql/apolloconfigdb.sql:348]: Done.

Comment (mghio) [reply scripts/sql/apolloconfigdb.sql:348]: > Please add a folder `v200-v210` in [delta](https://github.com/apolloconfig/apollo/tree/master/scripts/sql/delta) and add a upgrade script there.

Done.

**Review by nobodyiam — APPROVED** [review]
LGTM

**Pull request merged**